### PR TITLE
First set of changes to hydrogen data website

### PR DESCRIPTION
**Pull request by scjensen77**
I have added the project collaborators and some info to the home page for the hydrogen data project. This is only a start and needs to be improved before we submit the hydrogen collaboration data paper.

### CODE REVIEW

File: main.yaml [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
     name: what-wt
-    content: |
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-      commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
-      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-      est laborum.
+    content:
```
Comment (matthewturk) [inline main.yaml]: I think you might still need the `|` character here, to make it valid YAML.  You can check if it's valid YAML by trying to load it into python, with:

```python
import yaml
yaml.load(open("main.yaml", "r"), Loader=yaml.SafeLoader)
```
and seeing if it throws an error.

File: main.yaml [hunk 2]
```diff
@@ -1,48 +1,41 @@
...
+      Many aspects of the phase diagram of dense hydrogen remain poorly understood, 
+      sometimes even qualitatively. Dense hydrogen is predicted become a high-temperature 
+      superconductor at sufficiently high pressure and is crucial in determining the 
+      structure of gas giant planets. Addressing the entire phase diagram with accurate 
+      ab initio simulations like diffusion Monte Carlo (DMC) is not currently feasible
```
Comment (matthewturk) [inline main.yaml:9]: is diffusion usually lowercased, when prefacing an acronym?

File: main.yaml [hunk 3]
```diff
@@ -1,48 +1,41 @@
...
+
+    image: img/f70a_solid-phases.pdf
+  - title: yt Hub 
+    name: what-tale
+    content: |
+    image: img/ythub.png
+
+  - title: How to use the database
     name: what-tale
     content: |
```
Comment (matthewturk) [inline main.yaml:40]: I think you might want to leave this empty, rather than the `|`, which might confuse the parser.

File: main.yaml (hunk 1 above)
Comment (scjensen77) [reply main.yaml]: Oh interesting, thanks. When I was generating the website it did not seem to have issues but I didn't even realize the character was removed. We should leave it in as you suggest.

File: main.yaml (hunk 2 above)
Comment (scjensen77) [reply main.yaml:9]: I think the lowercase is okay here (just following the conventions we have been using in our other written texts). For instance, we use this in our recent work "Stable solid molecular hydrogen above 900K from a machine-learned potential trained with diffusion Quantum Monte Carlo".

File: main.yaml (hunk 3 above)
Comment (scjensen77) [reply main.yaml:40]: I am okay with what you suggest.

File: main.yaml (hunk 1 above)
Comment (matthewturk) [reply main.yaml]: For what it's worth, I was in fact incorrect!  It is not necessary to include the character.

**Pull request merged**